cloud/monitoring.rst

```diff
@@ -0,0 +1,39 @@
+Monitoring
+##########
+
+To monitor events in the life of a formation with outside tools via a standard format, we offer RSS feeds per organization. You can use a feed reader or RSS Slack integration (e.g. on an :code:`#ops` channel) to keep up to date.
+
+On the upper right of the "Formations" list in the Cloud console, follow the "Formation Events" link to the RSS feed. The feed includes entries for three types of events, each with the following details:
+
+**Server Unavailable**
+
+This is a notification of connectivity problems such as hardware failure.
+
+*  Formation name
+*  Formation url
+*  Server
+
+**Failover Scheduled**
+
+After a server is detected as unavailable Citus Cloud schedules a failover to a replacement.
```

Comment:
`After a server is detected as unavailable Citus Cloud schedules a failover to a replacement.` => `For planned upgrades, or when operating a formation without HA, this event will show to indicate a future planned failover event.`